lib/elephant_in_the_room_web/templates/login/login.html.eex

```diff
@@ -1,26 +1,36 @@
 <h2>Login Page</h2>
 
-<%= if @maybe_user == nil do %>
-  <%= form_for @changeset, @action, fn f -> %>
-    
-    <div class="form-group">
-      <%= label f, :username, class: "control-label" %>
-      <%= text_input f, :username, class: "form-control" %>
-      <%= error_tag f, :username %>
-    </div>
-    
-    <div class="form-group">
-      <%= label f, :password, class: "control-label" %>
-      <%= password_input f, :password, class: "form-control" %>
-      <%= error_tag f, :password %>
-    </div>
-
-    <div class="form-group">
-      <%= submit "Submit", class: "btn btn-primary" %>
-    </div>
-
-  <% end %>
-<% else %>
-  <h1>Hello <%= @maybe_user.username%>!</h1>
-  <span><%= link "Logout", to: login_path(@conn, :logout), method: :post %></span>
+<%= case @user do %>
+  <%= :no_user -> %>
+    <%= form_for @changeset, login_path(@conn, :login), fn f -> %>
+
+      <div class="form-group">
+        <%= label f, :username, class: "control-label" %>
+        <%= text_input f, :username, class: "form-control" %>
+        <%= error_tag f, :username %>
+      </div>
+
+      <div class="form-group">
+        <%= label f, :password, class: "control-label" %>
+        <%= password_input f, :password, class: "form-control" %>
+        <%= error_tag f, :password %>
+      </div>
+
+      <div class="form-group">
+        <%= submit "Submit", class: "btn btn-primary" %>
+      </div>
+      
+    <% end %>
+
+  <%= %ElephantInTheRoom.Auth.User{username: name} -> %>
```

Comment:
You can put an alias for `ElephantInTheRoom.Auth.User` to avoid writing the entire module name :)